add-kml-layer-with-network-links/README.metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+    "category": "Layers",
+    "description": "Display a file with a KML network link, including displaying any network link control messages at launch.",
+    "formal_name": "AddKmlLayerWithNetworkLinks",
+    "ignore": false,
+    "images": [
+        "add-kml-layer-with-network-links.png"
+    ],
+    "keywords": [
+        "KML",
+        "KMZ",
+        "OGC",
+        "keyhole",
+        "network link",
+        "network link control",
+        "KmlDataset",
+        "KmlLayer"
+    ],
+    "language": "kotlin",
+    "redirect_from": "TODO",
+    "relevant_apis": [
+        "KmlDataset",
+        "KmlLayer"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/arcgismaps/sample/addkmllayerwithnetworklinks/MainActivity.kt",
+        "src/main/java/com/esri/arcgismaps/sample/addkmllayerwithnetworklinks/components/SceneViewModel.kt",
```

Comment:
putting the scene model path first means it gets shown first in the sample viewer "code view". So we usually try to list viewmodel at the top of this list